tests/snuba/api/endpoints/test_organization_events.py

```diff
@@ -493,6 +495,132 @@ def test_auto_insert_project_name_when_event_id_present_with_aggregate(self):
             {"project.name": self.project.slug, "id": "a" * 32, "count()": 1}
         ]
 
+    def test_performance_issue_ids_filter(self):
+        _perf_group = self.create_group(type=GroupType.PERFORMANCE_N_PLUS_ONE_DB_QUERIES.value)
+
+        def hack_pull_out_data(jobs, projects):
+            _pull_out_data(jobs, projects)
+            for job in jobs:
+                job["event"].groups = [_perf_group]
+            return jobs, projects
+
+        data = load_data(
+            platform="transaction",
+            timestamp=before_now(minutes=10),
+            start_timestamp=before_now(minutes=11),
+        )
+        with mock.patch("sentry.event_manager._pull_out_data", hack_pull_out_data):
+            self.store_event(data=data, project_id=self.project.id)
+
+        query = {
+            "field": ["count()"],
+            "statsPeriod": "1h",
+            "query": f"performance.issue_ids:{_perf_group.id}",
+        }
+        response = self.do_request(query)
+        assert response.status_code == 200, response.content
+        assert response.data["data"][0]["count()"] == 1
```

Comment:
I don't know why but this test passes when I run just this function but fails when I run the entire suite

Reply:
this was due to `int(v, 16)` that was messing with group ids. this is resolved now